recipes/apriltag/all/conanfile.py

```diff
@@ -0,0 +1,53 @@
+import os
+import stat
+import shutil
+from conans import ConanFile, tools, CMake, AutoToolsBuildEnvironment
+from conans.errors import ConanException
+
+class apriltagConan(ConanFile):
+    name = "apriltag"
+    license = "BSD 2-Clause"
```

Comment:
```suggestion
    license = "BSD-2-Clause"
```

Use SPDX identifier https://spdx.org/licenses/